### ISSUE TITLE
Inbox Zero background does not work

### ISSUE THREAD

**Issue by CarlosRebolledo** [leggett/simplify#426]
Inbox Zero background does not work since when I get to Inbox Zero none of the funds to choose appear. Comes out blank

**Comment by leggett** [comment]
The inbox zero backgrounds only show up for two Gmail themes: the default white theme and the dark theme. In all other cases, I leave the background alone. For image themes this seems like the right thing to do. For simple color themes, it might make sense to figure out how to add an image in these cases too in a future release.

- Do you have a theme other than one of these two themes? If yes, it might help if you could post a screenshot of your window.
- Are you saying no background shows up? 
- What version of Simplify are you running? I published v2.1.14 last night which added some options around Inbox Zero backgrounds. Are these not working?

**Comment by CarlosRebolledo** [comment]
You're right. I was using the light gray theme and now that I test with the default Gmail theme the Inbox Zero backgrounds from Simplify v.1.14 look perfect.

Thank you very much Michael and have a great week
